rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/20_synonyms_get.yml

```diff
@@ -0,0 +1,129 @@
+setup:
```

Comment:
Do you think we should add a teardown here?

Reply:
... but we still don't have DELETE. I think it's better to wait for it and thus use the CRUD APIs for managing the tests setups and teardown.

Reply:
What `teardown` section can do here?

After each individual test section, we are supposed to delete all indices, including `.synonyms` index.